src/main/java/org/example/Main.java

```diff
@@ -0,0 +1,109 @@
+package org.example;
+
+import java.util.Scanner;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+import java.util.Map;
+import java.util.Random;
+
+import static org.example.Util.Result.*;
+
+
+public class Main {
+
+    private static final String randomWord = Util.getRandomWord();
+
+    private static int numberOfGuess = 0;
+    private static int totalNumberOfGuess = 6;
+
+
+    public static void main(String[] args) {
+
+        System.out.println(RED + "WELCOME" + BLUE + " TO A GAME " + CYAN + " OF WORDLE" + RESET);
+        while (numberOfGuess < totalNumberOfGuess) {
+            numberOfGuess++;
+            Scanner scanner = new Scanner(System.in);
+            System.out.println("enter any five letter word");
+            String word = scanner.nextLine();
+            boolean notTooLong;
+            notTooLong = word.length() == 5;
+            System.out.println(" Attempt " + numberOfGuess + " out of " + totalNumberOfGuess);
+
+            while (!notTooLong) {
+                System.out.println(RED + "Your guess has to be five letters" + RESET);
+                System.out.println("enter any five letter word");
+                word = scanner.nextLine();
+                notTooLong = word.length() == 5;
+
+            }
+            if (randomWord.equalsIgnoreCase(word)) {
+                System.out.println(PURPLE + "you Have guessed the word and won the game!" + RESET);
```

Comment:
System.out.println(PURPLE + "you have guessed the word and won the game!" + RESET);

Reply:
Fixed the grammatical error. Change capital H to h.